info/task_description.html

```diff
@@ -4,7 +4,7 @@
     You are given a string. Your function should return <b class="code">{% if is_js %}true{% else %}True{% endif %}</b> if the string is a valid number (contains only digits and "+-." at proper places), otherwise - <b class="code">{% if is_js %}false{% else %}False{% endif %}</b>. Look at the mask:
 </p>
 <pre>
-[+- ][zero or more digits][.][zero or more digits]
```

Comment:
It is not unnecessary.  In this mission the number may be "+34", "-34" and "34". The all are valid. If you remove whitespace, it means you must use either + or -, leaving no clue for just no-sign.